eventhub_client/devdoc/requirement_docs/eventhubclient_ll_requirements.md

```diff
@@ -43,7 +43,8 @@ DEFINE_ENUM(EVENTHUBCLIENT_STATE, EVENTHUBCLIENT_STATE_VALUES);
 #define EVENTHUBCLIENT_ERROR_RESULT_VALUES          \
     EVENTHUBCLIENT_SASTOKEN_AUTH_FAILURE,           \
     EVENTHUBCLIENT_SASTOKEN_AUTH_TIMEOUT,           \
-    EVENTHUBCLIENT_SOCKET_SEND_FAILURE              \
+    EVENTHUBCLIENT_SOCKET_SEND_FAILURE,             \
+    EVENTHUBCLIENT_SENDER_AMQP_UNINITIALIZED        \
```

Comment:
Instead of calling this EVENTHUBCLIENT_SENDER_AMQP_UNINITIALIZED, can you please call this  EVENTHUBCLIENT_AMQP_INIT_FAILURE?